src/test/java/finrem/cfv/CategoriesAssignDisplayOrder.java

```diff
@@ -11,16 +11,19 @@
 public class CategoriesAssignDisplayOrder {
 
     public void assignDisplayOrders() {
-        String filePath = "src/test/java/finrem/cfv/CategoriesToReassign.json";
+        String filePath = "definitions/contested/json/CaseTypeTab/CaseTypeTab.json";
```

Comment:
did you mean to change this class?

Reply:
I accidentally committed this then removed it